roop/core.py

```diff
@@ -223,8 +223,10 @@ def select_face_handler(path: str):
 
```

Comment:
This PR is nice but if you solve issues with face and target selection, it makes sense you take care of the Start and Preview button aswell - both should be disabled until valid face and target is present